coeus-impl/src/main/java/org/kuali/coeus/propdev/impl/person/ProposalDevelopmentPersonnelController.java

```diff
@@ -280,6 +281,12 @@ public ModelAndView certifyAnswers(@ModelAttribute("KualiForm") ProposalDevelopm
         return super.save(form);
     }
 
+    @Transactional @RequestMapping(value = "/proposalDevelopment", params = "methodToCall=retrieveCollectionPage")
+    public ModelAndView retrieveCollectionPage(@ModelAttribute("KualiForm") DocumentFormBase form, BindingResult result, HttpServletRequest request,
```

Comment:
So what about this method. I know the wizard is used in budget as well which is a different mapping. Do we need this there as well?